.github/workflows/publish-latest-v3.yml

```
@@ -0,0 +1,85 @@
+name: Publish to latest / v3
```

This shouldn't actually be publishing to latest though, right?
```suggestion
name: Publish to v3
```

---

You're right, adjusting